playground/src/main/scala/play/Playground.scala

```diff
@@ -0,0 +1,17 @@
+package play
+
+import com.github.limansky.mongoquery.scala_driver._
+import org.mongodb.scala.MongoCollection
+
+object Playground extends App {
```

Comment:
What is the purpose of this object? Is it gonna be an example application?

Reply:
Yep, it might be. I made it while implementing MSD (to test things up) and didn't delete it yet. But now I realize that it could be done on testing projects... So, should I remove it?

Reply:
I don't see any benefits having this here.

Reply:
Okay, removed.